Source/DataSources/dynamicGeometryGetBoundingSphere.js

```diff
@@ -25,12 +29,14 @@ define([
         //>>includeEnd('debug');
 
         var attributes;
+        var modelMatrix;
 
         //Outline and Fill geometries have the same bounding sphere, so just use whichever one is defined and ready
         if (defined(primitive) && primitive.show && primitive.ready) {
             attributes = primitive.getGeometryInstanceAttributes(entity);
             if (defined(attributes) && defined(attributes.boundingSphere)) {
-                BoundingSphere.transform(attributes.boundingSphere, primitive.modelMatrix, result);
+                modelMatrix = defaultValue(primitive.modelMatrix, Matrix4.IDENTITY);
```

Comment:
`GroundPrimitives` don't have a modelMatrix, they are always in the fixed frame.